addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -48,31 +48,36 @@ var LinkDialog = Dialog.extend({
         this.data.className = "";
         this.data.iniClassName = "";
 
-        const selection = editable.ownerDocument.getSelection();
-        this.data.range = selection.getRangeAt(0);
+        const selection = editable && editable.ownerDocument.getSelection();
```

Comment:
How can you end up with the link dialog but no editable area?

Reply:
Because it's extended and the extention does not use any editable. It just want to create a link and recieve it on save without adding it in the document directly.